public/app/plugins/datasource/jaeger/_importedDependencies/types/window.d.ts

```diff
@@ -0,0 +1,16 @@
+export declare global {
```

Comment:
Why do we need this file?

Reply:
I had a type error without this file but now it seems not needed anymore. Removed.